### DISCUSSION TITLE
Consolidate several system prompts in a single window

### DISCUSSION THREAD

**Discussion by Tobias-Artur** [SilentVoid13/Templater#447] [Help]
Hello there, 

I have researched this quite a bit, but I haven't found an answer to it.

Is it possible to consolidate several prompts, that come one after another, in a single window, similar to the Tasks Create/Edit plugin window:
![image](https://user-images.githubusercontent.com/86799266/142699078-5c95acbc-87eb-44c1-874e-769349731d84.png)

This is a snippet of my code:
```
const requester = await tp.system.prompt("Enter full name of the requester:")
const ticketNO = await tp.system.prompt("Enter ticket no:");
const line_start = await tp.system.prompt("Type in the title of the ticket");
```

Much appreciated

**Comment by AB1908** [comment]
I've gotten to this quite early apparently but would need to use Obsidian API for this. Take a look at some snippets from the plugins for ideas.
